jujugui/static/gui/src/app/components/user-profile/user-profile.js

```diff
@@ -130,19 +131,9 @@ YUI.add('user-profile', function() {
     */
     _interactiveLogin: function() {
       var bakery = this.props.charmstore.bakery;
-      bakery.fetchMacaroonFromStaticPath(this._fetchMacaroonCallback);
+      bakery.fetchMacaroonFromStaticPath(this.props.fetchMacaroonCallback);
```

Comment:
This `fetchMacaroonCallback` should be called from a local method because the local method will then need to make requests for things like the additional charms/models/etc.